test/commands/force/data/soql/query/dataSoqlQuery.nut.ts

```diff
@@ -155,6 +155,14 @@ describe('data:soql:query command', () => {
       );
       expect(queryResult).to.match(/Total number of records retrieved: 1\./g);
     });
+    it('should handle count()', () => {
+      const queryResult = execCmd('force:data:soql:query -q "SELECT Count() from User"', {
+        ensureExitCode: 0,
+      }).shellOutput as string;
+
+      expect(queryResult).to.match(/Total number of records retrieved: \d*\./g);
```

Comment:
wouldn't this still match the previous behavior of `Total number of records retrieved: 0.`